patches/4156082F - Marvel Ultimate Alliance 2.patch.toml

```diff
@@ -0,0 +1,13 @@
+title_name = "Marvel Ultimate Alliance 2"
+title_id = "4156082F" # AV-2095
+hash = "8BC78D3019FD8760" # default.xex
+#media_id = "00000000"
```

Comment:
Is this the media ID that showed up in the log?

Reply:
Caught in 4K 💀